src/utils/getInitials.ts

```diff
@@ -0,0 +1,11 @@
+export const getInitials = (name: string) => {
```

Comment:
Not really something that we probably need to take into account, but this makes the assumption that a person first name is the name that they are called by. So John Henry Doe becomes JD, but the persons calling name could be Henry. But I don't think that is something we need to take into account at this point :)

Reply:
Yep, might be something that needs to be checked in the future. This name will however come from the elasticsearch data, so we might need to do something there already.